### ISSUE TITLE
Avionics - update unit tests

### ISSUE THREAD

**Issue by lo9key** [OregonStateRocketry/30k2018-CS-Capstone#27]
We need to update our unit tests to get back up to >= 80%

The sensor classes remain mostly unchanged but may need updates.

- [x] mainPayloadState class needs tests.
- [x] payloadState class needs tests.

**Comment by lo9key** [comment]
As of tonight, here is the coverage report:

```
pi@esra-p:~/Payload $ coverage report -m ESCMotor.py MPL3115A2.py MPU9250.py PCF8523.py mainPayload.py payloadState.py
Name              Stmts   Miss  Cover   Missing
-----------------------------------------------
ESCMotor.py          56      2    96%   98, 100
MPL3115A2.py         23      0   100%
MPU9250.py           63      0   100%
PCF8523.py           25      4    84%   20, 38-46
mainPayload.py       41      7    83%   5, 10, 49, 90-96, 107-108
payloadState.py      94     61    35%   5, 10, 14, 19, 23, 49-56, 75-78, 85-97, 112-114, 123-132, 149-152, 161-170, 186-190, 197-211, 226, 229-232
-----------------------------------------------
TOTAL               302     74    75%
```

**Comment by lo9key** [comment]
End of the night report after commit 5c2c1b6d56255ff5490f11ff5c7b57a9002497fd:

```
pi@esra-p:~/Payload $ coverage report -m ESCMotor.py MPL3115A2.py MPU9250.py PCF8523.py mainPayload.py payloadState.py
Name              Stmts   Miss  Cover   Missing
-----------------------------------------------
ESCMotor.py          56      2    96%   98, 100
MPL3115A2.py         23      0   100%
MPU9250.py           63      0   100%
PCF8523.py           25      4    84%   20, 38-46
mainPayload.py       41      7    83%   5, 10, 49, 90-96, 107-108
payloadState.py      93      8    91%   10, 14, 19, 96-97, 225-228
-----------------------------------------------
TOTAL               301     21    93%
```

**Comment by lo9key** [comment]
Addressed by 07afa1b4d235b37f70317c7f8fbdd52901e023c5
